src/components/site/BreadCrumbLastItem.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const BreadcrumbLastItem = props => {
+  const { link, text } = props;
+  return (
+    <a className="breadcrumb breadcrumb-last" href={link}>
```

Comment:
as mentioned somewhere below i would like to try and avoid this component if possible. The only difference between this is a class, you can use the [classnames package](https://github.com/JedWatson/classnames) inside of `BreadcrumbLinkItem` to combine classes.  See [buttons](https://github.com/baltimorecounty/dotgov-components/blob/7f7d7733d3b03f8c567253603af62bb432f39c99/src/components/basic/Button.js#L7) for an example.

Let me know if you have questions about this.

Reply:
Makes sense and seems like the way to go. Removed this component in the most recent commit. Carrying the conversation back over to the BreadCrumbLinkItem.js component, where I have refactored.